vwifi.c

```diff
@@ -385,6 +447,13 @@ static struct owl_context *owl_create_context(void)
      */
     ret->wiphy->max_scan_ssids = 69;
 
+    /* Signal type
+     * CFG80211_SIGNAL_TYPE_UNSPEC allow us specify signal strength from 0 to
```

Comment:
Should be "allows"

Reply:
fixed on https://github.com/sysprog21/vwifi/pull/4/commits/0c6c0135e25ccf4e0a194724debba5862c1d8941